test-extra/Makefile

```diff
@@ -39,19 +41,23 @@ code/infer: URI = https://github.com/facebook/infer.git
 code/js_of_ocaml: URI = https://github.com/ocsigen/js_of_ocaml.git
 code/ocaml: URI = https://github.com/ocaml/ocaml.git
 code/dune: URI = https://github.com/ocaml/dune.git
-
-# DIRS += code/owl code/irmin code/index code/duniverse code/dune-release \
-# 	code/coq code/mirage code/alcotest code/citty
```

Comment:
This was commented on purpose to avoid spending too much time and power unless needed. This will make the script a lot slower.

Reply:
I don't see a huge difference on my side, this script is not supposed to be run as often as `dune runtest`, but we could group these repos in categories: community, mirage, janestreet for example and add an argument to decide on which groups to run the tests?

Reply:
It's still possible to remove packages before running, so it's still possible to do small runs.